services/community-site/src/pages/Delegation/delegation.scss

```diff
@@ -263,6 +263,15 @@ a.commissionChangeCheck {
 
     .validatorActions {
       float: right;
+
+      button {
+        border-radius: 8px;
+      }
+
+      .disabledButton {
```

Comment:
Don't need a class for this.
We can just use:
```
button:disabled,
button[disabled] {
}
```

Reply:
Fixed!